### PR TITLE
Fix: variable redeclaration in `subghz_cli.c`

### PR DESCRIPTION
**Pull request by derskythe**
# What's new

- Variable `i` was redeclared within an inner loop, potentially leading to undesirable behavior. To prevent any issues, this redeclaration was corrected by changing the inner loop's variable name to `j`.

# Verification 

- No undesirable behavior
 
# Checklist (For Reviewer)

- [ ] PR has description of feature/bug or link to Confluence/Jira task
- [ ] Description contains actions to verify feature/bugfix
- [ ] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: applications/main/subghz/subghz_cli.c [hunk 1]
```diff
@@ -1067,7 +1067,7 @@ static void subghz_cli_command_chat(Cli* cli, FuriString* args) {
                         furi_string_push_back(output, message[i]);
                         if(message[i] == '\n') {
                             printf("\r");
-                            for(uint8_t i = 0; i < 80; i++) {
+                            for(uint8_t j = 0; j < 80; i++) {
```
Comment (skotopes) [inline applications/main/subghz/subghz_cli.c]: To be honest this specific place was just shadowing previous declaration without causing any errors, but now `i++` increments completely different thing.

**Review by skotopes — CHANGES_REQUESTED** [review]

File: applications/main/subghz/subghz_cli.c (hunk 1 above)
Comment (derskythe) [reply applications/main/subghz/subghz_cli.c:1070]: @skotopes on the other hand, it still looks like a bug. 
But, if no one has any complaints, then it’s already a feature. 
But still I would add a comment with a link to the current pull request

Comment (skotopes) [reply applications/main/subghz/subghz_cli.c]: Just fix `i++` thing )

Comment (derskythe) [reply applications/main/subghz/subghz_cli.c]: @skotopes updated. As I understand this loop just create extra space to put line on the side.

File: applications/main/subghz/subghz_cli.c [hunk 2]
```diff
@@ -1064,16 +1064,26 @@ static void subghz_cli_command_chat(Cli* cli, FuriString* args) {
                     memset(message, 0x00, message_max_len);
                     size_t len = subghz_chat_worker_read(subghz_chat, message, message_max_len);
                     for(size_t i = 0; i < len; i++) {
-                        furi_string_push_back(output, message[i]);
                         if(message[i] == '\n') {
-                            printf("\r");
-                            for(uint8_t i = 0; i < 80; i++) {
+                            // Check it was \r before
+                            if((i > 0 && message[i - 1] != '\r') || i == 0) {
```
Comment (skotopes) [inline applications/main/subghz/subghz_cli.c:1069]: this time you overdid yourself: now layout is a total mess. Check with `./fbt cli`

**Review by skotopes — CHANGES_REQUESTED** [review]

File: applications/main/subghz/subghz_cli.c (hunk 2 above)
Comment (skotopes) [reply applications/main/subghz/subghz_cli.c:1069]: I mean what exactly you were trying to fix there?

**Pull request closed without merging**